__tests__/cmds/openapi/uses.test.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable jest/no-conditional-expect */
```

Comment:
`eslint-plugin-jest` doesn't like me doing `expect()` calls inside a `try-catch` and it wants me to instead write a wrapper for the command call and no thanks I'd rather not.

https://github.com/jest-community/eslint-plugin-jest/blob/v27.1.6/docs/rules/no-conditional-expect.md#how-to-catch-a-thrown-error-for-testing-without-violating-this-rule